exchanges/ftx/ftx_wrapper.go

```diff
@@ -0,0 +1,827 @@
+	var resp []order.Detail
+	for x := range getOrdersRequest.Pairs {
+		var tempResp order.Detail
+		orderData, err := f.GetOpenOrders(f.FormatExchangeCurrency(getOrdersRequest.Pairs[x], asset.Spot).String())
+		if err != nil {
+			return resp, err
+		}
+		for y := range orderData.Result {
+			tempResp.ID = strconv.FormatInt(orderData.Result[y].ID, 10)
+			tempResp.Amount = orderData.Result[y].Size
+			tempResp.AssetType = asset.Spot
+			tempResp.ClientOrderID = orderData.Result[y].ClientID
+			tempResp.Date = orderData.Result[y].CreatedAt
+			tempResp.Exchange = f.Name
+			tempResp.ExecutedAmount = orderData.Result[y].Size - orderData.Result[y].RemainingSize
+			tempResp.Pair = currency.NewPairFromString(orderData.Result[y].Market)
+			tempResp.Price = orderData.Result[y].Price
+			tempResp.RemainingAmount = orderData.Result[y].RemainingSize
+			var orderVars OrderVars
+			orderVars, err = f.compatibleOrderVars(orderData.Result[y].Side,
+				orderData.Result[y].Status,
+				orderData.Result[y].OrderType,
+				orderData.Result[y].FilledSize,
+				orderData.Result[y].Size,
+				orderData.Result[y].AvgFillPrice)
+			if err != nil {
+				return resp, err
+			}
+			tempResp.Status = orderVars.Status
+			tempResp.Side = orderVars.Side
+			tempResp.Type = orderVars.OrderType
+			tempResp.Fee = orderVars.Fee
+			resp = append(resp, tempResp)
+		}
+		triggerOrderData, err := f.GetOpenTriggerOrders(f.FormatExchangeCurrency(getOrdersRequest.Pairs[x], asset.Spot).String(), getOrdersRequest.Type.String())
+		if err != nil {
+			return resp, err
+		}
+		for z := range triggerOrderData.Result {
+			tempResp.ID = strconv.FormatInt(triggerOrderData.Result[z].ID, 10)
+			tempResp.Amount = triggerOrderData.Result[z].Size
+			tempResp.AssetType = asset.Spot
+			tempResp.Date = triggerOrderData.Result[z].CreatedAt
+			tempResp.Exchange = f.Name
+			tempResp.ExecutedAmount = triggerOrderData.Result[z].FilledSize
+			tempResp.Pair = currency.NewPairFromString(triggerOrderData.Result[z].Market)
+			tempResp.Price = triggerOrderData.Result[z].AvgFillPrice
+			tempResp.RemainingAmount = triggerOrderData.Result[z].Size - triggerOrderData.Result[z].FilledSize
+			tempResp.TriggerPrice = triggerOrderData.Result[z].TriggerPrice
+			orderVars, err := f.compatibleOrderVars(triggerOrderData.Result[z].Side,
+				triggerOrderData.Result[z].Status,
+				triggerOrderData.Result[z].OrderType,
+				triggerOrderData.Result[z].FilledSize,
+				triggerOrderData.Result[z].Size,
+				triggerOrderData.Result[z].AvgFillPrice)
+			if err != nil {
+				return resp, err
+			}
+			tempResp.Status = orderVars.Status
+			tempResp.Side = orderVars.Side
+			tempResp.Type = orderVars.OrderType
+			tempResp.Fee = orderVars.Fee
+			resp = append(resp, tempResp)
+		}
+	}
+	return resp, nil
+}
+
+// GetOrderHistory retrieves account order information
+// Can Limit response to specific order status
+func (f *FTX) GetOrderHistory(getOrdersRequest *order.GetOrdersRequest) ([]order.Detail, error) {
```

Comment:
I'm getting `FTX unsuccessful HTTP status code: 404 raw response: {"error":"No such market: TRYB/0626 ","success":false}` for this when the asset+pair type is for futures. 
Is it that order history doesn't work with Futures? 
If futures order history is unsupported, you should return an error when a futures asset is received to prevent calling FTX endpoint unnecessarily which ends the loop and returns the wrapper function.

But I imagine its because you are formatting the pair as spot, even if the pair is from futures. So you'll need to support futures pair formatting here. Potentially as an adjustment to `order.GetOrdersRequest` to include an asset type, or to check if the current currencies is a futures or spot request and formatting appropriately